src/components/ExpandCollapse/styles.module.scss

```diff
@@ -0,0 +1,7 @@
+.hideBorder {
```

Comment:
Maybe to rename the file to `ExpandCollapse.module.scss` or smth? Just to keep the convention